Assets/XML/Text/Technologies.xml

```diff
@@ -4118,15 +4413,19 @@
 	<TEXT>
 		<Tag>TXT_KEY_TECH_STATECRAFT</Tag>
 		<English>Statecraft</English>
-		<French>Statecraft</French>
+		<French>
+			<Text>Statecraft</Text>
```

Comment:
Really?

Reply:
Didn't know what to translate it to, skipped it, and forgot about it. Will try to find something.

Reply:
Maybe "Sciences administratives"?